### ISSUE TITLE
Error with first example on Nim 1.2.0

### ISSUE THREAD

**Issue by UNIcodeX** [liquid600pgm/rapid#19]
```
stack trace: (most recent call last)
gfx.nim(1033, 14)        loop
macros.nim(609, 24)      expectKind
c:\Users\unicodex\Desktop\code\sandbox\trapid\r1.nim(14, 8) template/generic instantiation of `loop` from here
c:\Users\unicodex\Desktop\code\sandbox\trapid\r1.nim(17, 10) Error: Expected a node of kind nnkStmtList, got nnkIdent

```

**Comment by liquid600pgm** [comment]
Yeah sorry, these examples are a little bit outdated. I kind of stopped developing rapid as much as I did in early 2019, but here's what's wrong:
The `loop` macro has been updated to make more sense. Previously it accepted an extra `step` variable for `update`, but since that that variable is pretty much useless in updates, I removed it.
Here's the example, corrected:
```nim
import rapid/gfx

var win = initRWindow()
  .title("My Game")
  .size(800, 600)
  .open()

var surface = win.openGfx()

surface.loop:
  draw ctx, step:
    ctx.clear(gray(255))
  update:
    discard
```
I'll update the readme to reflect this.
